src/js/elements/tutorial.js

```diff
@@ -0,0 +1,17 @@
+let wrapperClass = '.w-slider-mask';
```

Comment:
I would add these as class variables and not global variables.

Reply:
done